administrator/language/en-GB/com_content.ini

```
@@ -12,6 +12,10 @@ COM_CONTENT_ATTRIBS_ARTICLE_SETTINGS_LABEL="Options"
 COM_CONTENT_ATTRIBS_FIELDSET_LABEL="Options"
 COM_CONTENT_BATCH_OPTIONS="Batch process the selected articles"
 COM_CONTENT_BATCH_TIP="If a category is selected for move/copy, any actions selected will be applied to the copied or moved articles. Otherwise, all actions are applied to the selected articles."
+COM_CONTENT_BLANKSTATE_BUTTON_ADD="Add your first article"
+COM_CONTENT_BLANKSTATE_BUTTON_LEARNMORE="Learn more"
+COM_CONTENT_BLANKSTATE_CONTENT="Articles are the main content on most Joomla sites. You can use the Joomla Article Editor to create and manage your content."
+COM_CONTENT_BLANKSTATE_TITLE="Looks like you don't have any Articles yet"
```

```suggestion
COM_CONTENT_BLANKSTATE_TITLE="No Articles have been created yet."
```

The other strings are fine